iroh-ctl/src/main.rs

```diff
@@ -76,6 +84,11 @@ async fn main() -> anyhow::Result<()> {
         Commands::Status { watch } => {
             crate::status::status(client, watch).await?;
         }
+        Commands::Version => {
```

Comment:
This would be very useful having on each service. Issue is that the command would have to live on p2p and get imported from there to have the compile time env var work. Unsure if that's really worth it, but I can easily see a world where you're using iroh-ctl to address a number of instances from various versions.

Reply:
done!